sdk/anomalydetector/ai-anomaly-detector-rest/src/anomalyDetectorRest.ts

```diff
@@ -7,17 +7,17 @@ import { AnomalyDetectorRestClient } from "./clientDefinitions";
 
 /**
  * Initialize a new instance of the class AnomalyDetectorRestClient class.
- * @param Endpoint type: string Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus2.api.cognitive.microsoft.com).
- * @param ApiVersion type: string Anomaly Detector API version (for example, v1.1).
+ * @param Endpoint type: string
+ * @param ApiVersion type: "v1.1"
  * @param credentials type: KeyCredential
  */
 export default function createClient(
   Endpoint: string,
+  ApiVersion: "v1.1",
```

Comment:
From the cadl definition the ApiVersion is an extensible enum not enum

- Why do we implement it as an enum?
- How to allow other values inputted like: `v1.2`?

```
enum Versions {
  v1_1: "v1.1",
}

@knownValues(Versions)
model APIVersion is string;
```